app/services/document_capture_session_result.rb

```diff
@@ -26,9 +26,9 @@ def selfie_status
     self[:selfie_status].to_sym
   end
 
-  def success_status
+  def success_status(selfie_required:)
```

Comment:
this seems like a really unexpected place to pass a param, sepecially since this method is aliased as `:success?` as well, does that mean we need to update every callsite to pass the param? or add a default value?

is there any way we can pass the `selfie_required` bit as part of the data that is stored inside this struct so the callees don't have to pass this in?